.github/workflows/pr.yaml

```diff
@@ -44,3 +44,33 @@ jobs:
 
       - name: ${{ matrix.engine }}-build
         run: make ${{ matrix.engine }}-build
+
+  release-build:
+    runs-on: ubuntu-latest
+    services:
+      registry:
+        image: registry:2
+        ports:
+          - 5000:5000
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup Go
+        uses: actions/setup-go@v4
+        with:
+          go-version-file: 'go.mod'
+
+      - name: Install operator-sdk
+        # Instructions from https://sdk.operatorframework.io/docs/installation/#install-from-github-release
+        run: |
+          export ARCH=$(case $(uname -m) in x86_64) echo -n amd64 ;; aarch64) echo -n arm64 ;; *) echo -n $(uname -m) ;; esac)
+          export OS=$(uname | awk '{print tolower($0)}')
+          export OPERATOR_SDK_DL_URL=https://github.com/operator-framework/operator-sdk/releases/download/v1.33.0
+          curl -LO ${OPERATOR_SDK_DL_URL}/operator-sdk_${OS}_${ARCH}
+          chmod +x operator-sdk_${OS}_${ARCH} && sudo mv operator-sdk_${OS}_${ARCH} /usr/local/bin/operator-sdk
+
+      - name: release-build
```

Comment:
do we need to build the catalog source in this pr check when we're ALSO building it in https://github.com/janus-idp/operator/blob/main/.github/workflows/pr-docker-build.yaml#L95

Reply:
Oh right. It is indeed not needed here in that case. I wanted to prevent the specific issue from happening again. So to do that, we'll need to reuse the `make release-build` in https://github.com/janus-idp/operator/blob/main/.github/workflows/pr-docker-build.yaml?rgh-link-date=2024-01-03T15%3A14%3A02Z#L98-L102
I'll revert the change in this file. Thanks.

Reply:
Done in [`2dd4a8a` (#112)](https://github.com/janus-idp/operator/pull/112/commits/2dd4a8aa80e22a3eafd30724bd35986896b029ed)